lib/commands/alert.js

```diff
@@ -130,21 +137,27 @@ helpers.getAlert = async function getAlert () {
     const okButton = possibleAlertButtons[1];
     await assertButtonName(okButton, 'ok');
 
+    // add cancel function to the alert, clicking the 'Cancel' button
     alert.cancel = async () => {
       await this.proxyCommand(`/element/${util.unwrapElement(cancelButton)}/click`, 'POST');
     };
+    // add ok function to the alert, clicking the 'OK' button
     alert.ok = async () => {
       await this.proxyCommand(`/element/${util.unwrapElement(okButton)}/click`, 'POST');
     };
   }
 
+  // add getText function to the alert, getting the value of the correct element
   alert.getText = async () => {
-    let textView = await this.findNativeElementOrElements('class name', 'XCUIElementTypeTextView', false, util.unwrapElement(alert));
-    return await this.proxyCommand(`/element/${util.unwrapElement(textView)}/attribute/value`, 'GET');
+    // iOS up to 13.3 will report a single text view, while 13.4 will have two
+    // but the _last_ one will be the one presenting the text of the alert
+    const textViews = await this.findNativeElementOrElements('class name', 'XCUIElementTypeTextView', true, util.unwrapElement(alert));
+    return await this.proxyCommand(`/element/${util.unwrapElement(_.last(textViews))}/attribute/value`, 'GET');
   };
+  // add setText function to the alert, setting the value of the text field element
   alert.setText = async (value) => {
     try {
-      let textView = await this.findNativeElementOrElements('class name', 'XCUIElementTypeTextField', false, util.unwrapElement(alert));
+      const textView = await this.findNativeElementOrElements('class name', 'XCUIElementTypeTextField', false, util.unwrapElement(alert));
       await this.proxyCommand(`/element/${util.unwrapElement(textView)}/value `, 'POST', {value});
     } catch (err) {
       if (isErrorType(err, errors.NoSuchElementError)) {
```

Comment:
this could be simplified by finding multiple matches and checking their count line in the helper above rather than catching the exception and checking its type